src/main/resources/modules/colorectal_cancer.json

```diff
@@ -948,7 +948,83 @@
         "high": 7,
         "unit": "days"
       },
-      "direct_transition": "Oncologist_Encounter"
+      "complex_transition": [
+        {
+          "condition": {
+            "condition_type": "Attribute",
+            "attribute": "colorectal_cancer_stage",
+            "operator": "==",
+            "value": 1
+          },
+          "distributions": [
+            {
+              "transition": "Check_Anemia_Exist",
+              "distribution": 0.4
+            },
+            {
+              "transition": "Oncologist_Encounter",
+              "distribution": 0.6
+            }
+          ]
+        },
+        {
+          "distributions": [
+            {
+              "transition": "Check_Anemia_Exist",
+              "distribution": 0.53
+            },
+            {
+              "transition": "Oncologist_Encounter",
+              "distribution": 0.47
+            }
+          ],
+          "condition": {
+            "condition_type": "Attribute",
+            "attribute": "colorectal_cancer_stage",
+            "operator": "==",
+            "value": 2
+          }
+        },
+        {
+          "distributions": [
+            {
+              "transition": "Check_Anemia_Exist",
+              "distribution": 0.66
+            },
+            {
+              "transition": "Oncologist_Encounter",
+              "distribution": 0.34
+            }
+          ],
+          "condition": {
+            "condition_type": "Attribute",
+            "attribute": "colorectal cancer stage",
```

Comment:
This attribute is `colorectal cancer stage` but needs to be `colorectal_cancer_stage`